dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -1090,6 +1092,8 @@ class DevtoolsStartupTest {
         device.deviceId,
       ]);
 
+      await device.awaitDevice();
```

Comment:
this looks like the right place. In case someone adds more code in the future that also has an uninstall, maybe we should make a wrapper method:

```dart
class Device {
  Future<void> uninstallApp(String deviceId) async {
      await flutter('install', options: <String>[
        '--uninstall-only',
        '-d',
        deviceId,
      ]);

      await awaitDevice();
  }
}
```

Reply:
Ah yeah. That is a good idea! I'll do that.